java/src/jmri/jmrit/logixng/actions/DigitalFormula.java

```diff
@@ -393,6 +395,20 @@ public void unregisterListenersForThisClass() {
     public void disposeMe() {
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public void getUsageDetail(int level, NamedBean bean, List<NamedBeanUsageReport> report, NamedBean cdl) {
+        log.debug("getUsageReport :: Action DigitalFormula: bean = {}, f = \"{}\", cdl = {}", bean, getFormula(), cdl);
+        if (bean != null) {
+            // Search for possible bean references using system and user names.  False positives are possible.
+            var formula = getFormula();
+            var uname = bean.getUserName();
+            if (uname == null) uname = "9zQqtVv7";
+            if (formula.contains(bean.getSystemName()) || formula.contains(uname)) {
```

Comment:
I think it would be better if these two lines are changed to:

if (formula.contains(bean.getSystemName()) || (uname != null && formula.contains(uname))) {